drivers/usbhost/usbhost_enumerate.c

```diff
@@ -51,7 +51,9 @@ static void usbhost_putle16(uint8_t *dest, uint16_t val);
 static inline int usbhost_devdesc(const struct usb_devdesc_s *devdesc,
               FAR struct usbhost_id_s *id);
 static inline int usbhost_configdesc(const uint8_t *configdesc, int desclen,
-              FAR struct usbhost_id_s *id);
+                                     const uint8_t start_ifnum,
```

Comment:
```suggestion
                                     uint8_t start_ifnum,
```

Reply:
> 
Please give a reason for removing const on what is supposed to be a readonly variable.

Reply:
@xiaoxiang781216  I agree with @ldube, if anything the other parameters here should also have `const`.

Reply:
It's debatable whether the const non pointer parameters should be added const or not, but the whole code base seldom(never?) add const to this type of parameter. I am natural to add const or not for this case, but the same rule must be applied universally to the whole code base. So I would suggest to create a new PR to add const to all const parameter.

Reply:
Sure let's remove const for consistency.